internal/settings/ovh.go

```diff
@@ -142,3 +169,77 @@ func (o *ovh) Update(ctx context.Context, client *http.Client, ip net.IP) (newIP
 		return nil, fmt.Errorf("%w: %s", ErrUnknownResponse, s)
 	}
 }
+
+func (o *ovh) updateWithZoneDNS(client *ovhClient.Client, ip net.IP) (newIP net.IP, err error) {
+	recordType := A
+	var ipStr string
+	if ip.To4() == nil { // IPv6
+		recordType = AAAA
+		ipStr = ip.To16().String()
+	} else {
+		ipStr = ip.To4().String()
+	}
+	// subDomain filter of the ovh api expect an empty string to get @ record
+	subDomain := o.host
+	if subDomain == "@" {
+		subDomain = ""
+	}
```

Comment:
How about `*`? 🤔

Reply:
star `*` works, I don't know why `@` doesn't. Instead, we have to pass an empty string to `&subdomain=` argument. And if we do not supply this argument there is no filter (except fieldType) and every records (except those filtered out by fieldType) are returned. So basically the default I coded is returned `@` unless another subdomain (`host`) is set